
#User
class UserExistsError(Exception):
    pass

class UserNotFoundError(Exception):
    pass
